### DISCUSSION TITLE
Connecting b-parasite through Zigbee router

### DISCUSSION THREAD

**Discussion by ZehelSitchel** [rbaron/b-parasite#151] [Q&A]
I have 10 b-parasite boards running the Zigbee firmware. I want to move them away form the coordinator but any time I move them away they just disconnect. Except for 1 that connected to 1 of the 4 Third Reality Smart Plugs that are also Zigbee routers.

I've re-flashed them, set them directly next to the same smart plug as the other 1 has connected to, pulled the battery for 10 seconds then reconnected, factory reset by bridging gnd and rst 2 times, removed them from Zigbee2MQTT, and re-paired.   

Is anyone else using a router with their b-parasites? Any recommendations to get them to connect to a router instead of the coordinator?

My coordinator is the HomeAssistant Sky Connect.

<img width="669" alt="Screenshot 2023-07-16 192121" src="https://github.com/rbaron/b-parasite/assets/6528720/36ce02ea-b0c5-437b-955e-ea3114012923">

**Comment by rbaron** [comment]
Hi @ZehelSitchel. I have a few of them around the house, and they seem to work okay and (re-)connect to routers, even after I move them to different rooms. Note that they may take a while (O(minutes) maybe?)  to reconnect, as they are a low duty sleepy end device. For example:

<img width="846" alt="image" src="https://github.com/rbaron/b-parasite/assets/1573409/3188163d-6c03-47ac-b48c-8feacc278ca6">

I'm running ZHA + SkyConnect.

**Reply by ZehelSitchel** [reply]
Maybe I'm being to aggressive and need to connect and move them one at a time.

I'd hate to have to roll back to ZHA, but I'll give that a go as well. Could be something with Zigbee2MQTT.
